addons/mrp/static/src/scss/mrp_fields.scss

```
@@ -1,3 +1,8 @@
 .o_field_widget.o_embed_url_viewer iframe {
     aspect-ratio: 3/2;
 }
+
+.overflow-scrollable {
+        overflow-x: auto;
+        scrollbar-width: thin;
```

this should be 4 space indent

```suggestion
    overflow-x: auto;
    scrollbar-width: thin;
```